README.md

```diff
@@ -6,416 +6,198 @@
 [![CI-Nightly](https://github.com/JuliaSIMD/Polyester.jl/actions/workflows/CI-julia-nightly.yml/badge.svg)](https://github.com/JuliaSIMD/Polyester.jl/actions/workflows/CI-julia-nightly.yml)
 [![Coverage](https://codecov.io/gh/JuliaSIMD/Polyester.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/JuliaSIMD/Polyester.jl)
 
-Note that `Polyester.@batch` moves arrays to threads by turning them into [StrideArraysCore.PtrArray](https://github.com/JuliaSIMD/StrideArraysCore.jl)s.
-This means that under an `@batch` slices will create `view`s by default, and that you may also need to start Julia with `--check-bounds=yes` while debugging.
+Polyester.jl provides **low-overhead multithreading** in Julia. The primary API is `@batch`, which can be used to parallelize for-loops (similar to `@threads`).
+
+Polyester implements static scheduling (c.f. `@threads :static`) and has minimal overhead because it manages and re-uses a dedicated set of Julia tasks. This can lead to (great) speedups compared to other multithreading variants (see [Benchmark]() below).
+
+## Basic usage example
 
-Polyester.jl provides low overhead threading.
-The primary API is `@batch`, which can be used in place of `Threads.@threads`.
-The number of available threads is still governed by [`--threads` or `JULIA_NUM_THREADS`](https://docs.julialang.org/en/v1.6/manual/multi-threading/#Starting-Julia-with-multiple-threads), as reported by `Threads.nthreads()`.
-Lets look at a simple benchmark.
 ```julia
-using Polyester, LinearAlgebra, BenchmarkHistograms
-# Single threaded.
-function axpy_serial!(y, a, x)
-    @inbounds for i in eachindex(y,x)
-        y[i] = muladd(a, x[i], y[i])
+using Polyester
+
+function axpy_polyester!(y, a, x)
+    @batch for i in eachindex(y,x)
+        y[i] = a * x[i] + y[i]
     end
 end
-# One thread per core, the default (the threads are not pinned)
-function axpy_per_core!(y, a, x)
-    @batch per=core for i in eachindex(y,x)
-        y[i] = muladd(a, x[i], y[i])
+
+a = 3.141
+x = rand(10_000)
+y = rand(10_000)
+axpy_polyester!(y, a, x)
+```
+
+## Important Notes
+
+* `Polyester.@batch` moves arrays to threads by turning them into [StrideArraysCore.PtrArray](https://github.com/JuliaSIMD/StrideArraysCore.jl)s. This means that under an `@batch` slices will create `view`s by default(!). You may want to start Julia with `--check-bounds=yes` while debugging.
```

Comment:
[No automatic `@inbounds` as of `StrideArraysCore.jl` 0.5](https://github.com/JuliaSIMD/StrideArraysCore.jl/pull/43)
but using `--check-bounds=yes` is good advice, anyway.
Especially because `@batch` adds `@inbounds`.

Reply:
Not sure I understand.
* Will arrays this be turned into `PtrArray`?
* Will slices still default to `view`?

I'll add a note about `@batch` automatically adding `@inbounds`.

(I have to say I'm not a fan of adding automatic `@inbounds` and defaulting to `views` inside of `@batch`. For me, the latter is about low-overhead threading. The former two are optimizations that a user can add themself if they want to. But it's your package.)